### ISSUE TITLE
Change HOST constant to a variable in analyzer and maven plugin

### ISSUE THREAD

**Issue by aggenebbisj** [sdaschner/jaxrs-analyzer#19]
In the ```SwaggerBackend``` the host is a constant:

``` java
public class SwaggerBackend implements Backend {

    private static final String SWAGGER_VERSION = "2.0";
    private static final String HOST = "example.com";
```
So it always ends up in the json as:

``` json
{"swagger":"2.0","info":{"version":"0.1-SNAPSHOT","title":"project"},"host":"example.com", ... }
```

If you use the swagger ui the "try it out!" feature will use ```example.com```. It would be nice if you could specify the host through a maven configuration variable.

**Comment by sdaschner** [comment]
Sounds good.
The name and the version of the project still have to be taken from maven as well.

**Comment by aggenebbisj** [comment]
Nice to hear :-) Nice project btw.

Had a quick attempt at creating a pull request, but had to comment out the GPG license stuff to get the maven build working. If you like, I could show the code (it is a very quick attempt though). On the other hand, this should be quite trivial, right?

**Comment by sdaschner** [comment]
Yes, sure. Feel free to PR ;-)

**Comment by sdaschner** [comment]
I've changed the backends to include the project name, version and deployed domain (which is taken as swagger host). Added in e866cd7 and included in the latest v0.4 version.
Thanks for your contribution!
